gateway/src/main/java/discord4j/gateway/intent/Intent.java

```diff
@@ -0,0 +1,127 @@
+package discord4j.gateway.intent;
+
+public enum Intent {
+
+    /**
+     * Events which will be received by subscribing to GUILDS
```

Comment:
I definitely see the benefit to including this information in our docs, but I'm on the fence about it because we aren't always the best at updating information copied from Discord's docs whenever it changes as we have to closely track all changes to the docs (see Routes).

Even if we do decide to include them, I think it would be more beneficial to link to our actual `discord4j.core.event.domain` classes.